bare-metal/container-linux/kubernetes/profiles.tf

```diff
@@ -1,18 +1,39 @@
+locals {
+  flavor  = "${element(split("-", var.os_channel), 0)}"
+  channel = "${element(split("-", var.os_channel), 1)}"
+
+  profile      = "${local.flavor == "flatcar" ? "fl" : "cl"}"
```

Comment:
Sorry. I was unclear in my review. By separate profile, I was referring to separate Matchbox Profiles. A Matchbox profile defines the initrd, kernel, args, etc. that should be used by matched machines. I was proposing instead of using variables throughout the "container-linux-install" "matchbox_profile", I'd consider defining a separate "flatcar-linux-install" profile.

Wasn't proposing making separate directories for cl and fl